Content.Client/Roles/RoleTimerSystem.cs

```diff
@@ -0,0 +1,73 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using Content.Shared.CCVar;
+using Content.Shared.Roles;
+using Robust.Client.Player;
+using Robust.Shared.Network;
+using Robust.Shared.Players;
+
+namespace Content.Client.Roles;
+
+public sealed class RoleTimerSystem : SharedRoleTimerSystem
+{
+    [Dependency] private readonly IPlayerManager _playerManager = default!;
+
+    public TimeSpan OverallPlaytime { get; private set; }
+    private Dictionary<string, TimeSpan> _roles = new();
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeNetworkEvent<RoleTimersEvent>(OnRoleTimers);
```

Comment:
This is out-of-simulation info, it should be managed on a regular manager via `NetMessage`s, not entity messages.